helm-chart/openj9-jitserver-chart/Chart.yaml

```diff
@@ -0,0 +1,42 @@
+###############################################################################
+# Copyright (c) 2020, 2020 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+apiVersion: v2
+name: openj9-jitserver-chart
+description: |-
+  Eclipse OpenJ9 JITServer Helm Chart. 
+
+  License
+  By installing this product you accept the Eclipse Public License v2.0: https://www.eclipse.org/legal/epl-2.0/.
```

Comment:
The chart is EPLv2 but the binaries installed by it are under GPLv2+CE as they are combined with the OpenJDK class libraries.  Which license is being called out here?

Reply:
Thanks for catching that. The license should actually be consistent with the rest part of the codebase. Is the below text a proper way to explain the license? 

```
License
By installing this product you accept the GNU General Public License, 
version 2 with the GNU Classpath Exception [1] and GNU General Public 
License, version 2 with the OpenJDK Assembly Exception [2].

[1] https://www.gnu.org/software/classpath/license.html
[2] http://openjdk.java.net/legal/assembly-exception.html
```